README.md

```diff
@@ -1,167 +1,19 @@
-# Scope: what and why
+# Variable Scope: Lab
 
-Imagine if all parts of your code knew about all the variables in all other parts of your code. You'd have to take great care not to accidentally reuse a variable name in multiple places -- `point` may be an x-y coordinate in one place but lat-long elsewhere, and you'd have to keep that all in mind. Yikes!
+## Objectives
 
-Luckily, the visibility of classes, methods, and variables is controlled by what is called **scope**. The things that a particular piece of code can see are called "in scope". Those it cannot are "out of scope".
+1. Use method arguments to pass values horizontally between contexts.
+2. Write loops and conditionals with an awareness of scope.
+3. Pull a value out of a localized context by declaring it prior to the localization.
```

Comment:
"localization" and "localized" mean something different (support for other languages) -- again, not a thing that's in the vernacular in this sense.

I would make this more general -- "Understand how variables come into and go out of scope"